samples/generated/snippets.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to do basic operations using Cloud
+Spanner.
+For more information, see the README.rst under /spanner.
+"""
+
+import time
+
+from google.cloud import spanner
+from google.cloud.spanner_admin_instance_v1.types import spanner_instance_admin
+
+OPERATION_TIMEOUT_SECONDS = 240
+
+
+# [START spanner_create_instance]
+def create_instance(instance_id):
+    """Creates an instance."""
+    spanner_client = spanner.Client()
+
+    config_name = "{}/instanceConfigs/regional-us-central1".format(
+        spanner_client.project_name
+    )
+
+    operation = spanner_client.instance_admin_api.create_instance(
```

Comment:
We should not rely on `instance_admin_api` because we should remove this once we ask customers to start using autogenerated admin clients.

Reply:
> instance_admin_api
No, this is not true, we will never remove spanner.Client support.

Reply:
We will just ask custmer to not use spanner.Client() exposed methods for admin operations and eventually the methods will be only removed we will never remove `spanner_client.instance_admin_api`.